package.json

```diff
@@ -4,7 +4,7 @@
   "description": "MyDiary is an online journal where users can pen down their thoughts and feelings.",
   "main": "index.js",
   "scripts": {
-    "dev": "node server/index.js --exec babel-node",
+    "dev": "npm run build && node dist/index.js --exec babel-node",
```

Comment:
Is there a reason why you should transpile your project.

Reply:
yes for it to run in node otherwise it won't run